cpp/src/column/column_view.cpp

```diff
@@ -175,6 +179,52 @@ mutable_column_view::operator column_view() const
   return column_view{_type, _size, _data, _null_mask, _null_count, _offset, std::move(child_views)};
 }
 
+void const* column_view::get_data() const noexcept
+{
+  std::string key{"column_view::get_data"};
+  if (cudf::experimental::prefetch::detail::PrefetchConfig::instance().get(key)) {
+    if (cudf::is_fixed_width(type())) {
+      cudf::experimental::prefetch::detail::prefetch_noexcept(
+        key, _data, size() * size_of(type()), cudf::get_default_stream());
+    } else if (type().id() == type_id::STRING) {
+      strings_column_view scv{*this};
+
+      cudf::experimental::prefetch::detail::prefetch(
```

Comment:
This should be `prefetch_noexcept`. Fixed locally, will push after build completes.